File: src/IsbInterop.CSharp/Accessory/Wrappers/SystemInfo.cs
namespace IsbInterop.Accessory.Wrappers
{
  /// <summary>
  /// Обертка над ISystemInfo.
  /// </summary>
  internal class SystemInfo : BaseIsbObject, ISystemInfo
  {
    /// <summary>
    /// Версия клиента.
    /// </summary>
    public string ClientVerson => (string)GetRcwProperty("ClientVersion");

    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="systemInfo">Объект с информацией о системе.</param>
    /// <param name="scope">Область видимости.</param>
    internal SystemInfo(object systemInfo, IScope scope) : base(systemInfo, scope) { }
  }
}

File: src/IsbInterop/Accessory/Proxies/SystemInfo.vb
Namespace Accessory.Proxies

  ''' <summary>
  ''' Обертка над ISystemInfo.
  ''' </summary>
  Friend Class SystemInfo
    Inherits BaseIsbObject
    Implements ISystemInfo

    ''' <summary>
    ''' Версия клиента.
    ''' </summary>
    Public ReadOnly Property ClientVerson As String Implements ISystemInfo.ClientVerson
      Get
        Return DirectCast(GetRcwProperty("ClientVersion"), String)
      End Get
    End Property

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="systemInfo">Объект с информацией о системе.</param>
    ''' <param name="scope">Область видимости.</param>
    Friend Sub New(systemInfo As Object, scope As IScope)
      MyBase.New(systemInfo, scope)
    End Sub
  End Class
End Namespace
